src/scripts/modules/media/body/content-baked.less

```diff
@@ -178,8 +185,6 @@ h1.example-title .text {
 .note     { .make-block(note); }
 [data-type="example"],
 .example  { .make-block(example); }
-[data-type="abstract"],
-.abstract { .make-block(note); }
```

Comment:
This looks like some global mixin for `.abstract`. Should we remove it or just reset in `.os-chapter-outline`? @helenemccarron